backend/sidekiq_config.md

```diff
@@ -0,0 +1,130 @@
+##Background Jobs using Sidekiq
+
+This document outlines the configuration required for a Rails application using the Sidekiq gem for background job processing to be deployed on Heroku.
+
+For detailed information, refer to:
+
+
+* [https://github.com/mperham/sidekiq/wiki/Getting-Started]()
+* [https://devcenter.heroku.com/articles/redistogo]()
+* [https://devcenter.heroku.com/articles/scheduler]()
+
+
+###Rails Configuration
+
+Include the sidekiq gem in the ```Gemfile``` and run install bundle.
+
+```
```

Comment:
Coupla markdown tips:

* Leave a space b/w headers (##) and title associated with it
* For inline code formatting (vs. a block of code) you can use single backticks (```) instead of triple
* To get nice syntax highlighting, you can type the name of the language after the triple backticks (e.g. "```ruby")